### ISSUE TITLE
Blank screen problem

### ISSUE THREAD

**Issue by Costiny** [lazydevyo/PSVitaDock#6]
The program was working fine until I downloaded the newest v4 Vitadock. Now every time I connect my Vita, the screen on the program just goes blank, no matter the settings. I also tried both builds (my pc is 64-bit).

**Comment by Draqen88** [comment]
the same happens in the 32-bit version, blank screen.

**Comment by lazydevyo** [comment]
Which psvita version u have? 1000 or 2000?

**Comment by ExtremeCaliber** [comment]
Same problem here unfortunately. I've tried both builds even though my system is x64, and it's a white screen. Though, I can still hear my line in, but the screen is just plain white. I tried the previous version, and it worked perfectly fine which is odd. I'm using the psp slim 2001

**Comment by lazydevyo** [comment]
Download the debugger.. tell me if the debugger recognize the vita, and if not tell me which devices you get.

**Comment by ExtremeCaliber** [comment]
I just tried it out, It seems to have found the vita but I can't click on the check mark since it went behind the "X". 
![image](https://user-images.githubusercontent.com/41549730/53022622-8ddec080-3429-11e9-89d2-021c9c518667.png)

**Comment by ExtremeCaliber** [comment]
Opening the debugger whilst the vita is connected yields the same result but clicking on the check mark does nothing
![image](https://user-images.githubusercontent.com/41549730/53022702-b8307e00-3429-11e9-9351-d6d0800a7ca8.png)

**Comment by lazydevyo** [comment]
> Opening the debugger whilst the vita is connected yields the same result but clicking on the check mark does nothing
> ![image](https://user-images.githubusercontent.com/41549730/53022702-b8307e00-3429-11e9-9351-d6d0800a7ca8.png)

Yeah the checkmark doesnt do anything by cliking it, you use the debugger only to check if the streaming plugin detects the vita, since it detects it the psvitadock should work normaly, so after using the debugger and getting a positive you should use the psvitadock now.

**Comment by ExtremeCaliber** [comment]
I'll try it out rn!

**Comment by ExtremeCaliber** [comment]
This is what happens on both 64 and 32 bit versions. The previous version (v3.2) works fine though

![image](https://user-images.githubusercontent.com/41549730/53031663-4a8d4d80-343b-11e9-917a-21765fbcc455.png)
^ (v4)

![image](https://user-images.githubusercontent.com/41549730/53031916-cc7d7680-343b-11e9-8c2b-cca0ef6b14f5.png)

^ (v3.2)

**Comment by lazydevyo** [comment]
I only get white screen if something is blocking the camera access to psvitadock.

**Comment by ExtremeCaliber** [comment]
Did you mean camera access to use the mic? Because I can hear my own mic from my webcam on both versions.

**Comment by lazydevyo** [comment]
> Did you mean camera access to use the mic? Because I can hear my own mic from my webcam on both versions.

How you hear your own mic from your webcam does your webcam has speakers

**Comment by Costiny** [comment]
I'm having exactly the same 2 results as ExtremeCaliber. Using PSV Fat model.

**Comment by Costiny** [comment]
> 
> 
> > Did you mean camera access to use the mic? Because I can hear my own mic from my webcam on both versions.
> 
> How you hear your own mic from your webcam does your webcam has speakers

He means the webcam has an integrated speaker. I had the same issue, but honestly I haven't tried again connectin with the jack, I prefer just using the vita speakers.

**Comment by ExtremeCaliber** [comment]
Yeah I have a line in port that I would've used if it worked. I tried it with v3.2 and it was a blast. Yeah my webcam has an integrated mic.

**Comment by ExtremeCaliber** [comment]
I just tried unplugging my webcam and it actually worked

I'm surprised that this was the problem

**Comment by ExtremeCaliber** [comment]
Though, I think it would be more convenient if I could leave my webcam plugged in. I tried it with my laptop since that was my original intention on using vitadock, so then I can see it on the big screen. But, it's the same white screen.. and I can't exactly unplug my integrated webcam and mic from my laptop xD. unless I remove the device or something

**Comment by lazydevyo** [comment]
> Though, I think it would be more convenient if I could leave my webcam plugged in. I tried it with my laptop since that was my original intention on using vitadock, so then I can see it on the big screen. But, it's the same white screen.. and I can't exactly unplug my integrated webcam and mic from my laptop xD. unless I remove the device or something

Don't worry ill work on it.

**Comment by ExtremeCaliber** [comment]
Thanks man, I appreciate it!

**Comment by Draqen88** [comment]
> Which psvita version u have? 1000 or 2000?

Model 1000 3G, henkaku enso in fw 3.60, plugin in use: nonpdrm, ReF00D

**Comment by Ac3dUd3** [comment]
same issue for me vita 1000, white screen but can hear sound

**Comment by psv1kdump** [comment]
i got exacetly same problem with the ExtremeCaliber

**Comment by Costiny** [comment]
Any news?

**Comment by ExtremeCaliber** [comment]
What's weird is that after you disable the device, start the app, and renable your webcam, then it works perfectly fine. haven't done further testing but that's just what I've been doing to get passed it

**Comment by Costiny** [comment]
Guess it shouldn't be much problem solving that yeah

**Comment by Zero2028** [comment]
I have the same problem. Just white screen(on psvitaDock prog.)

**Comment by srgtsilent** [comment]
I can confirm another case of the white screen issue.
It is solved by disabling your webcam video driver in processes for the time being.

**Comment by Zero2028** [comment]
@srgtsilent Thanks! Thats work for me!

**Comment by jopjob** [comment]
https://www.youtube.com/watch?v=_I_HgDQzj_0&t=1s <--- Fix the whitescreen

**Comment by zatenzu** [comment]
I have to disable DroidCam drivers to solve it. Start -> Device manager -> Sound, video and game controller -> right click on DroidCamSource & DroidCam Virtual Audio -> Select disable

**Comment by TheCyberQuake** [comment]
Honestly at this point I would just use OBS. Can't be bothered to constantly play around with removing all of my other video interfaces given I had several and one of which being PCIe.
